inst/shiny/modules/tab_visuals.R

```diff
@@ -97,19 +97,19 @@ tab_visuals_ui <- function(id) {
         sidebar = sidebar(
           selectInput(
             inputId = ns("analyte_mean"),
-            label = "Choose the Analyte:",
+            label = "Select Analyte:",
             choices = NULL,
             multiple = TRUE
           ),
           selectInput(
             inputId = ns("pcspec_mean"),
-            label = "Choose the Matrix:",
+            label = "Select Matrix:",
```

Comment:
suggestion: this one seems more intuitive with the column name
```suggestion
            label = "Select Specimen:",
```